<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterSettings</class>
 <widget class="QWidget" name="FilterSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>525</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QGridLayout">
     <property name="margin">
      <number>4</number>
     </property>
     <property name="spacing">
      <number>4</number>
     </property>
     <item row="3" column="0">
      <spacer name="Spacer9">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="bAddFilter">
       <property name="toolTip">
        <string>Add configured filter</string>
       </property>
       <property name="whatsThis">
        <string>Press this button to add the configured filter to the list.</string>
       </property>
       <property name="text">
        <string>A&amp;dd</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="bRemoveFilter">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Remove selected filter</string>
       </property>
       <property name="whatsThis">
        <string>Press this button to remove the selected filter from the list.</string>
       </property>
       <property name="text">
        <string>R&amp;emove</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="3">
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>Filter Properties</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0" colspan="2">
         <widget class="QComboBox" name="comboFilterAction">
          <property name="toolTip">
           <string>Action for this filter</string>
          </property>
          <property name="whatsThis">
           <string>Here you can define what should happen if this filter matches (e.g. whether the matching articles should be shown or hidden).</string>
          </property>
          <item>
           <property name="text">
            <string>Hide</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Show</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="lArticles">
          <property name="text">
           <string>articles whose headlines</string>
          </property>
          <property name="wordWrap">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3">
         <widget class="QComboBox" name="comboFilterCondition">
          <property name="toolTip">
           <string>Condition for this filter</string>
          </property>
          <property name="whatsThis">
           <string>This combo box lets you specify the condition under which the keyword/expression you entered in the input field at the right will match. You can select one of the following values:&lt;ul&gt;
&lt;li&gt;contain - the filter matches if the headline contains the keyword.&lt;/li&gt;
&lt;li&gt;does not contain - the filter matches if the headline does not contain the keyword.&lt;/li&gt;
&lt;li&gt;equals - the filter matches if the headline equals the expression.&lt;/li&gt;
&lt;li&gt;does not equal - the filter matches if the headline does not equal the expression.&lt;/li&gt;
&lt;li&gt;matches - the filter matches if the expression matches the headline. The expression you typed at the right will be considered a regular expression in this mode.&lt;/li&gt;</string>
          </property>
          <item>
           <property name="text">
            <string>contain</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>do not contain</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>equal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>do not equal</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>match</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="0" column="4">
         <widget class="QLineEdit" name="leFilterExpression"/>
        </item>
        <item row="1" column="2" colspan="3">
         <widget class="QComboBox" name="cboNewsSources"/>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>in</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item row="1" column="0" colspan="3">
      <widget class="QTreeWidget" name="filterEntries">
       <column>
        <property name="text">
         <string>Action</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Condition</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Expression</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Affects</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <tabstops>
  <tabstop>bAddFilter</tabstop>
  <tabstop>bRemoveFilter</tabstop>
 </tabstops>
 <includes>
  <include location="global">klineedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
